import_flow_events.py

```diff
@@ -84,18 +130,81 @@
     CREDENTIALS 'aws_access_key_id={aws_access_key_id};aws_secret_access_key={aws_secret_access_key}'
     FORMAT AS CSV;
 """
-
-Q_CHECK_FOR_DAY = """
-    SELECT timestamp FROM flow_events
-    WHERE timestamp >= {timestamp}
-    AND timestamp < {timestamp} + 86400
-    LIMIT 1;
+Q_INSERT_METADATA = """
+    WITH durations AS (
+      SELECT flowId, MAX(flowTime)
+      FROM flow_csv
+      GROUP BY flowId
+    )
+    INSERT INTO flow_metadata (
+      flowId,
+      beginTime,
+      duration,
+      completed,
+      newAccount,
+      browser,
+      browserVersion,
+      os,
+      context,
+      entrypoint,
+      migration,
+      service,
+      utmCampaign,
+      utmContents,
+      utmMedium,
+      utmSource,
+      utmTerm
+    )
+    SELECT (
+      begin.flowId,
+      (begin.timestamp * 1000.0),
+      (durations.flowTime * 1000.0),
```

Comment:
Yeah, I agree, having them as proper timestamp columns is the Right Thing even if it makes the import a bit weird.